dashboards/user.jsonnet

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env jsonnet -J ../vendor
+// Deploys one dashboard - "JupyterHub dashboard",
+// with useful stats about usage & diagnostics.
```

Comment:
Seems like a remnant of a copy paste operation

Reply:
@consideRatio fixed!